parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -170,13 +170,20 @@ $global:EnableHostsConfigAgent = [System.Convert]::ToBoolean("{{ EnableHostsConf
 # These scripts are used by cse
 $global:CSEScriptsPackageUrl = "{{GetVariable "windowsCSEScriptsPackageURL" }}";
 
+# The windows nvidia gpu driver related urls are used by windows cse
+$global:GpuDriverURL = "{{GetVariable "windowsGpuDriverURL" }}";

```

Comment:
what is the endpoint of the urls? We need to assure that they can be accessed in different clouds with custom's firewall settings.

Reply:
[Pull Request 8878419](https://dev.azure.com/msazure/CloudNativeCompute/_git/aks-rp/pullrequest/8878419): test agentbaker (this is a draft)

The url can be set differently for different clouds. Currently in the rp PR draft, they are all set to be the same urls. Those urls don't work on the private clouds. 
RP PR: [https://dev.azure.com/msazure/CloudNativeCompute/_git/aks-rp/pullrequest/8878419?_a=files&path=/resourceprovider/server/microsoft.com/datamodel/model/aksengine_azenvtypes.go](url)

Chris and I are currently working on creating a pipeline to release the drivers to acs mirror. Then we will change the url on the rp side to make sure they work on private clouds.